samples/3d-platformer/build.gradle.kts

```diff
@@ -4,11 +4,11 @@ plugins {
 }
 
 repositories {
-    mavenLocal()
```

Comment:
Why no mavenLocal?

Reply:
There's no need for it.

Reply:
hmm but it would be if it were a separate project. If i understand your composite build setup correctly, the `build.gradle.kts` (but not `settings.gradle.kts` but there it's commented) from the samples files look the same regardless if we are using composite builds or not. So i would leave them as they are even if they contain unused or redundant parts as they serve the purpose of being examples of how you can setup a project.
Or we can comment it.

That said, I'm nitpicking here. I'm fine either way :-)

Reply:
Even without composite build, `mavenLocal` is not needed because we don't expect users to build the library first - but used the either the actual release or the dev release.